tests/unit/src/test_conditionsStore.cpp

```diff
@@ -225,11 +225,9 @@ SCENARIO( "Creating a ConditionsStore", "[ConditionsStore][Creation]" )
 			{
 				REQUIRE( 0 == store.Get("ungreeted world") );
 				REQUIRE( primarySize(store) == 2 );
-				REQUIRE( store.Get("ungreeted world") == 0 );
-				REQUIRE( primarySize(store) == 2 );
 				REQUIRE( 0 == store.Get("hi world") );
 				REQUIRE( primarySize(store) == 2 );
-				REQUIRE( store.Get("hi world") == 0 );
+				REQUIRE( 0 == store.Get("hi world") );
 				REQUIRE( primarySize(store) == 2 );
```

Comment:
These are also duplicated and above you removed the duplicate checks. Are the duplicates check there to verify that the symmetry works?

Reply:
No, the check is that accessing the condition twice also doesn't give bad effects.
Some older implementations of ConditionsStore used caching which could affect 2nd and later lookups (while not affecting initial lookups). I'll add a comment for this.